packages/posthog/src/definitions.ts

```diff
@@ -144,6 +144,27 @@ export interface PosthogPlugin {
    * @since 6.0.0
    */
   unregister(options: UnregisterOptions): Promise<void>;
+  /**
+   * Opt in to event capturing.
+   *
+   * @since 7.5.0
+   */
+  optIn(): Promise<void>;
```

Comment:
Please sort these methods alphabetically by method name. This also applies to the android/ios/web implementation.